zk-token-sdk/src/instruction/transfer_with_fee.rs

```diff
@@ -384,7 +371,28 @@ impl ZkProofData<TransferWithFeeProofContext> for TransferWithFeeData {
     }
 }
 
-// #[derive(Clone, Copy, Pod, Zeroable)]
+#[allow(non_snake_case)]
+#[cfg(not(target_os = "solana"))]
+impl TransferWithFeeProofContext {
+    fn new_transcript(&self) -> Transcript {
+        let mut transcript = Transcript::new(b"transfer-with-fee-proof");
+        transcript.append_message(b"ciphertext-lo", bytes_of(&self.ciphertext_lo));
+        transcript.append_message(b"ciphertext-hi", bytes_of(&self.ciphertext_hi));
+        transcript.append_message(
+            b"transfer-with-fee-pubkeys",
+            bytes_of(&self.transfer_with_fee_pubkeys),
+        );
+        transcript.append_message(
+            b"new-source-ciphertext",
+            bytes_of(&self.new_source_ciphertext),
+        );
+        transcript.append_message(b"fee-ciphertext-lo", bytes_of(&self.fee_ciphertext_lo));
+        transcript.append_message(b"fee-ciphertext-hi", bytes_of(&self.fee_ciphertext_hi));
+        transcript.append_message(b"fee-parameters", bytes_of(&self.fee_parameters));
```

Comment:
It looks like this wasn't present before. Is it needed?

Reply:
Yep, before, the individual components of the `fee_ciphertext_lo` and `fee_ciphertext_hi` were translated into bytes and appended to the transcript. Now, I simplified it by just converting the entire `fee_ciphertext` and `fee_ciphertext_hi` into bytes and appending it to transcript.

Reply:
Oh sorry, I meant `fee-parameters` field specifically. Was that somehow included before? I couldn't see it

Reply:
Ah yes, sorry I forgot to mention it in the PR description. It is okay to not append `fee-parameters`, so we did not have it before, but I thought it could be safer to just append the entire context info, which includes the fee parameters. The transcript is a hash, so this does not increase the proof size of anything like that.

Reply:
Ah gotcha, thanks for the explanation!